learn/app-development/services/mock-services/mock-imported-apis.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Mocking Imported REST APIs"
+id: ""
+sidebar_label: "Mocking REST APIs"
+---
+---
+
+## Mocking Imported REST APIs
+WaveMakers new feature to mock imported REST API by one click. For an outage of your backend APIs due to some X reasons, go with WaveMaker mock features where the mocked server will simulate APIs using the same Imported swagger with sample responses,  these sample responses are dummy data with close-to-accurate responses giving the application a real feel with refreshed responses every time.
```

Comment:
@apitive this needs to be worded differently. Can you use the justification sentences i shared with you some time ago instead?